django/db/migrations/operations/models.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, name, fields, options=None, bases=None, managers=None):
         self.options = options or {}
         self.bases = bases or (models.Model,)
         self.managers = managers or []
+        name = name.strip('\"\'')
```

Comment:
The escaping shouldn't be on the model name but on it's `Meta.db_table` option. You also can't simply strip quotes from `options['db_table']` for multiple reasons.

You'll want to make the changes in the SQLite schema editor as advised in the ticket because right now this will not prevent the re-making of the following model's table from crashing

```python
class Foo(models.Model):
    class Meta:
        db_table = '"table_with_quoted_name"'
```

Which is the crux of the issue because Python doesn't allow defining classes containing quotes in the first place.